rdbms/src/test/java/io/spine/server/storage/jdbc/aggregate/CloseablesTest.java

```diff
@@ -20,60 +20,45 @@
 
 package io.spine.server.storage.jdbc.aggregate;
 
-import com.google.common.collect.Lists;
-import com.google.common.collect.Sets;
-import com.google.common.testing.NullPointerTester;
+import com.google.common.collect.ImmutableSet;
 import io.spine.server.storage.jdbc.aggregate.given.CloseablesTestEnv.FaultyClosable;
 import io.spine.server.storage.jdbc.aggregate.given.CloseablesTestEnv.StatefulClosable;
+import io.spine.testing.UtilityClassTest;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import java.util.Collection;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.stream.IntStream;
 
-import static io.spine.testing.DisplayNames.HAVE_PARAMETERLESS_CTOR;
-import static io.spine.testing.DisplayNames.NOT_ACCEPT_NULLS;
-import static io.spine.testing.Tests.assertHasPrivateParameterlessCtor;
+import static com.google.common.collect.Lists.newArrayList;
+import static com.google.common.collect.Sets.newHashSet;
+import static com.google.common.truth.Truth.assertThat;
 import static java.util.Collections.singleton;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.instanceOf;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@DisplayName("Closeables utility should")
-class CloseablesTest {
+@DisplayName("`Closeables` utility should")
+class CloseablesTest extends UtilityClassTest<Closeables> {
 
-    @Test
-    @DisplayName(HAVE_PARAMETERLESS_CTOR)
-    void haveUtilityConstructor() {
-        assertHasPrivateParameterlessCtor(Closeables.class);
-    }
-
-    @Test
-    @DisplayName(NOT_ACCEPT_NULLS)
-    void passNullToleranceCheck() {
-        new NullPointerTester()
-                .testAllPublicStaticMethods(Closeables.class);
+    CloseablesTest() {
+        super(Closeables.class);
     }
 
-    @SuppressWarnings("MethodWithMultipleLoops")
-    // Two loops - one for data set up and one for checks.
     @Test
     @DisplayName("close all passed instances")
     void closeAll() {
-        int count = 10;
-        Set<StatefulClosable> closeables = new HashSet<>(count);
-        for (int i = 0; i < count; i++) {
-            closeables.add(new StatefulClosable());
-        }
+        ImmutableSet<StatefulClosable> closeables = IntStream
+                .of(10)
+                .mapToObj(i -> new StatefulClosable())
+                .collect(ImmutableSet.toImmutableSet());
 
         Closeables.closeAll(closeables);
 
-        for (StatefulClosable sc : closeables) {
```

Comment:
While I understand this is a test code, I don't understand why change a perfectly readable piece of code into something significantly less comprehensible, which also creates an extra `Stream` instance :)

Please consider switching from `stream().map().forEach()` back to a simple foreach loop.

And yes, it was better with a warning suppression, if that's a reason. The problem here is that this approach may be eventually get copied for many times, and not just to the test code.